starter-code/iteration3/iteration3/src/app/app.component.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+
+<h1>NBA Championships</h1>
+
+<div *ngFor= "let winner of championships" >
+  <h1>{{winner.year}}</h1>
+  <h2>{{winner.champion}}</h2>
+  <img src="{{winner.championUrl}}" alt="">
+  <h3>Best players</h3>
+  <ul>
+    <li *ngFor= "let index of [0,1,2]">
+      <span>{{winner.bestPlayers[index]}}</span>
+    </li>
+  </ul>
```

Comment:
You can iterate the array of best players with the ngFor. Remember that ```player``` is a name that I decided to give to each item of the array on each iteration.
```
<ol>
    <li *ngFor= "let player of winner.bestPlayers">
      <span>{{player}}</span>
    </li>
</ol>
```